app/pages/homepage.py

```diff
@@ -23,20 +23,48 @@ def homepage_layout(user_headers, user_credentials):
-                                style={"position": "relative"},
+                                id="zoom-containement-container",
+                                className="common-style",
+                                style={"overflow": "hidden"},
                                 children=[
                                     html.Div(
-                                        id="image-container",
+                                        id="image-container-with-bbox",
+                                        style={"position": "relative"},
                                         children=[
-                                            html.Img(
-                                                src="./assets/images/no-alert-default.png",
-                                                className="common-style",
-                                                style={"max-width": "100%", "height": "auto"},
-                                            )
+                                            html.Div(
+                                                id="image-container",
+                                                children=[
+                                                    html.Img(
+                                                        id="main-image",
+                                                        src="./assets/images/no-alert-default.png",
+                                                        className="zoomable-image",
+                                                        style={"maxWidth": "100%", "height": "auto"},
+                                                    )
+                                                ],
+                                            ),
+                                            html.Div(
+                                                id="bbox-container",
+                                                style={"display": "block"},
+                                                children=[
+                                                    html.Div(
+                                                        id="bbox-positioning",
+                                                        style={"display": "none"},
+                                                        children=[
+                                                            html.Div(
+                                                                id="bbox-styling",
+                                                                style={
+                                                                    "border": "2px solid red",
+                                                                    "height": "100%",
+                                                                    "width": "100%",
+                                                                    "zIndex": "10",
+                                                                },
```

Comment:
This breakdown between `bbox-positioning` and `bbox-styling` elements is a hacky way to divide between those styling aspects that are changed by Dash updates, and those that are changed by JavaScript.

In this case, positioning is changed by Dash but JS needs a way to access `border` separately to reduce the width of the bbox as the picture is zoomed onto (in a way that sticks regardless of changes to the positioning).